### ISSUE TITLE
func checkRootless always write NSlog

### ISSUE THREAD

**Issue by DG12** [IngmarStein/Monolingual#88]
checkRootless should always write to NSlog, if SIP  enabled  or not.

Actually checkRootless really checks create and update access to directory 
`/System/Library/Monolingual.sip`
see #84 and #87 

Thank you.

**Comment by IngmarStein** [comment]
The result of `checkRootless()` is already printed to the console as a debug message (in `Helper.init()`.

It's just an implementation detail that a temporary directory is created. Please let me know if you're aware of other ways to determine SIP status. Last time I looked, there was no public API (https://forums.developer.apple.com/message/46849#46849).

**Comment by Eitot** [comment]
I do not find anything new about it. I suppose other options are:
- pipe `/usr/bin/csrutil`,
- pipe `/usr/sbin/nvram` and read the variable (if present),
- process all files regardless and ignore write errors,
- add an advanced preference and let users enable it themselves.

**Comment by DG12** [comment]
It seems that the question is not really in SIP is on,  rather can files be deleted or modified.
  I just enabled SIP and ran Monolingual.
   I find many messages like:
Sep 23 21:37:29 smackerpro Monolingual[38302]: Blacklisting com.alsoft.diskwarrior
   in the syslog warning level file, ending with :
Sep 23 21:37:29 smackerpro Monolingual[38302]: Blacklisting com.rossetantoine.osirix
Sep 23 21:37:29 smackerpro Monolingual[38302]: Adding root /Applications
Sep 23 21:37:29 smackerpro Monolingual[38302]: Adding root /Library
Sep 23 21:37:29 smackerpro Monolingual[38302]: Excluding root /System
Sep 23 21:37:29 smackerpro Monolingual[38302]: Excluding root /bin
Sep 23 21:37:30 smackerpro com.github.IngmarStein.Monolingual.Helper[53730]: MonolingualHelper started
Sep 23 21:40:08 smackerpro Monolingual[38302]: helper finished with exit code: 0
Sep 23 21:40:08 smackerpro com.github.IngmarStein.Monolingual.Helper[53730]: exiting with exit status 0
Sep 23 21:40:08 smackerpro Monolingual[38302]: Closing connection to helper
Sep 23 21:40:08 smackerpro Monolingual[38302]: XPC connection to helper invalidated.

    Notice that there is no message regarding rootless, and the app ran for 2.5 minutes. 

   It did update : ~/Library/Logs/Monolingual.log with
Monolingual started at 9/23/16, 9:51 PM
Removing languages: ach.lproj ...,...,  many...
   then
Deleted files:
 no files listed as deleted, and that's the end.

Thanks again for your concern.
